src/Host/Broker/Impl/Extensions/ExceptionExtensions.cs

```diff
@@ -2,15 +2,47 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System;
+using System.IO;
+using System.Runtime.ExceptionServices;
+using Microsoft.AspNetCore.Hosting;
 using Microsoft.AspNetCore.Server.Kestrel.Internal.Networking;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.R.Host.Protocol;
 
-namespace Microsoft.R.Host.Broker {
+namespace Microsoft.R.Host.Broker.Extensions {
```

Comment:
I prefer to avoid 'Extensions' in namespace so all methods naturally appear in the completion list as long as main definition is available.